src/Protocol/Protocol_1_9.cpp

```diff
@@ -2524,7 +2524,15 @@ void cProtocol_1_9_0::HandlePacketBoatSteer(cByteBuffer & a_ByteBuffer)
 void cProtocol_1_9_0::HandlePacketChatMessage(cByteBuffer & a_ByteBuffer)
 {
 	HANDLE_READ(a_ByteBuffer, ReadVarUTF8String, AString, Message);
-	m_Client->HandleChat(Message);
+	if ((Message.size()/8) > MAX_CHAT_MSG_LENGTH)
```

Comment:
Why is message size divided by 8 here? Surely this would result in the max length being 2048, as per your commit message.

Reply:
I just made it so beacuse Message.size() returns the size of the message in bits. So a size() call on a 256 character message returns 2048. I divide by 8 here so that the value of MAX_CHAT_MSG_LENGTH can correspond to the size in bytes/ amount of characters

Reply:
But Message is just a String, calling size() on it should return bytes rather than bits. Maybe I'm missing something here.

Reply:
No I was missing something. I was also confused as to why .size() would return bytes. Turns out, it did not. I was using a python script that wrote messages to the server as a bot, and thought I was configuring the number of characters it sends. I was actually configuring the number of times it adds an 8 byte string to the message it is going to send. Nevermind, I fixed that up and increased the MAX_STRING_SIZE to 1024, as 256 was too little.